README.md

```diff
@@ -1,11 +1,7 @@
 # @loopback/graphql
 
-LoopBack extension for [GraphQL](http://graphql.org/).
+The official GraphQL extension for LoopBack was moved to
+[`loopback-next`](https://github.com/strongloop/loopback-next) monorepo,
+directory [`extensions/graphql`](https://github.com/strongloop/loopback-next/tree/c12d926cbfb2e7bbb1e4c7827baad879eb3db097/extensions/graphql).
```

Comment:
do we want to use the `master` branch instead? i.e. 
https://github.com/strongloop/loopback-next/tree/master/extensions/graphql

Reply:
Sure :+1: